releases/8.1/languages/pt_BR.php

```diff
@@ -0,0 +1,91 @@
+    'performance_jit_arm64' => 'Backend JIT para ARM64 (AArch64)',
+    'performance_inheritance_cache' => 'Cache de herança (evita religamento de classes em cada requisição)',
+    'performance_fast_class_name_resolution' => 'Resolução de nome de classe rápida (evita conversão em minúsculas busca de hash)',
+    'performance_timelib_date_improvements' => 'Melhorias de performance na timelib e ext/date',
+    'performance_spl' => 'Melhorias em iteradores de sistema de arquivo SPL',
+    'performance_serialize_unserialize' => 'Otimizações em serialize/unserialize',
+    'performance_internal_functions' => 'Otimização de algumas funções internas (get_declared_classes(), explode(), strtr(), strnatcmp(), dechex())',
+    'performance_jit' => 'Melhorias e correções no JIT',
+
+    'other_new_title' => 'Novas Classes, Interfaces e Funções',
+    'other_new_returntypewillchange' => 'Novo atributo <code>#[ReturnTypeWillChange]</code>.',
+    'other_new_fsync_fdatasync' => 'Novas funções <code>fsync</code> e <code>fdatasync</code> functions.',
+    'other_new_array_is_list' => 'Nova função <code>array_is_list</code>.',
+    'other_new_sodium_xchacha20' => 'Novas funções Sodium XChaCha20.',
+
+    'bc_title' => 'Depreciações e quebras de retrocompatibilidade',
+    'bc_null_to_not_nullable' => 'Passagem de null para parâmetros não anuláveis de funções internas está depreciada.',
+    'bc_return_types' => 'Tipos de retorno provisórios em métodos de classes embutidas do PHP.',
+    'bc_serializable_deprecated' => 'Interface <code>Serializable</code> depreciada.',
+    'bc_html_entity_encode_decode' => 'Funções de de/codificação de entidades HTML processam aspas simples e substituem por padrão.',
+    'bc_globals_restrictions' => 'Restrições da variável <code>$GLOBALS</code>.',
+    'bc_mysqli_exceptions' => 'MySQLi: Modo de erro padrão definido para exceções.',
+    'bc_float_to_int_conversion' => 'Conversão inconpatíveis de float para int implícitas está depreciada.',
+    'bc_finfo_objects' => 'Extenção finfo: Recursos <code>file_info</code> migrados para objectos finfo existentes.',
+    'bc_imap_objects' => 'IMAP: Recursos imap migrados para objetos da classe <code>IMAP\Connection</code>.',
+    'bc_ftp_objects' => 'Extensão FTP Extension: Recursos de conexão migrados para objetos da classe <code>FTP\Connection</code>.',
+    'bc_gd_objects' => 'Extensão GD: Identificadores de fonte migrados para objetos da classe <code>GdFont</code>.',
+    'bc_ldap_objects' => 'LDAP: Recursos migrados para objetos <code>LDAP\Connection</code>, <code>LDAP\Result</code>, e <code>LDAP\ResultEntry</code>.',
+    'bc_postgresql_objects' => 'PostgreSQL: Recursos migrados para <code>PgSql\Connection</code>, <code>PgSql\Result</code>, e <code>PgSql\Lob</code> objects.',
+    'bc_pspell_objects' => 'Pspell: Recursos pspell e pspell config migrados para objetos de classe <code>PSpell\Dictionary</code> e <code>PSpell\Config</code>.',
```

Comment:
```suggestion
    'bc_pspell_objects' => 'Pspell: Configurações de recursos pspell e pspell migrados para objetos de classe <code>PSpell\Dictionary</code> e <code>PSpell\Config</code>.',
```

Reply:
Aque eu deixei a palavra "config" intencionalmente já que o sentido original da frase é indicar que as funções pspell_* trabalhavam com dois tipos de recursos diferentes:
- pspell
- pspell config

E estes foram migrados, respectivamente, para as classes:
- PSpell\Dictionary
- PSpell\Config

https://github.com/php/web-php/blob/e61897e327990fb137603291223ea836b797fad7/releases/8.1/languages/en.php#L79

O que acha de traduzirmos assim:

```suggestion
    'bc_pspell_objects' => 'Pspell: Recursos de dicionário e configuração migrados para objetos de classe <code>PSpell\Dictionary</code> e <code>PSpell\Config</code>.',
```

Reply:
Gostei, Lucas. +1